src/servises/filterTodos.ts

```diff
@@ -0,0 +1,15 @@
+import { Todo } from '../types/Todo';
+
+export const filteredTodos = (todos: Todo[], filter: string) => {
+  return todos.filter(todo => {
+    if (filter === 'active') {
```

Comment:
```suggestion
    if (filter === 'active') {
```
Consider creating an enum for statuses